services/QuillLMS/lib/query_examples/admin_diagnostics/diagnostic-students-view.sql

```diff
@@ -0,0 +1,82 @@
+        /*
+           Data Processed By Query: 1.5 GB
+           Bytes Billed For Query:  0.0 GB
+           Total Query Time:        4030 ms
```

Comment:
Note: a previous run of this benchmarking script (with no code changes), ran at `7521ms`.  We'll want to do a more robust examination of actual performance here to understand average and max execution times.